### DISCUSSION TITLE
Working set of Kubernetes Manifest files

### DISCUSSION THREAD

**Discussion by mitchtchesnitch** [ludeknovy/jtl-reporter#238] [Show and tell]
Thank you very much for making this great tool available!

I've taken some time to translate the docker-compose.yaml to kubernetes manifest files, to the best of my abilities. It's by no means optimal, but it does appear to work without any issues, and should be a good starting point for anyone looking to spin this up quickly on a cluster.

You can find the files here https://github.com/mitchtchesnitch/Jtl-Reporter-K8s/tree/main/Kubernetes_Manifests

Some notes/particularities:

-For the database init script, I am using a lifecycle postStart that downloads the schema.sql and applies if if the db structures are not already there.

The database manifest is also tailored to my setup, AWS EKS, using the default GP2 storageclass and the AWS EBS csi driver, you may have to alter the storageclass pointer to best suit your needs.

-The backend and migration are rolled into a singular pod, the migration is configured as an InitContainer, runs once before the main BE container.

-The ingress is again specific to my setup, as I am using the AWS Load Balancer controller, which allows me to create AWS LB components based on ingress annotations, I don't expect this to work outside this context.

Furthermore, I have an annotation that triggers the external dns controller to create a dns record in AWS Route53 for my ingress.

Tagging @ludeknovy. I hope this saves someone some time down the line, thank you very much for your effort, tool works great!

**Comment by ludeknovy** [comment]
Hi @mitchtchesnitch ! Thanks a lot. I am sure this will help some folks!
